client/my-sites/domains/domain-management/list/options-domain-button.jsx

```diff
@@ -119,14 +119,11 @@ class AddDomainButton extends Component {
 
 		let label = translate( 'Other domain options' );
 		if ( specificSiteActions ) {
-			label = translate( 'Add a domain' );
+			label = translate( 'Add new domain' );
```

Comment:
Do we have "Add new domain" in translations already?  Since this component is used in production we could do something like `hasTranslation( 'Add new domain' ) || englishLocales.includes(locale) ? translate( 'Add new domain' ) : translate( 'Add a domain' )` to ensure translated assets continue to be delivered for i18n users. That or hold the PR on translation freeze 😬

Reply:
Looks like we've already committed this text at https://github.com/Automattic/wp-calypso/blob/e1bfd881a5f5bb83e5950186d519c7220b87b847/client/hosting-overview/components/active-domains-card.tsx#L33

Reply:
String appears to be translated in April on https://translate.wordpress.com/